scripts/run.sh

```diff
@@ -270,21 +281,10 @@ run_simulator() {
   --priority-fee=1
 }
 
-if [[ ${SKIP_NETWORK_RUNNER_START} == false ]]; then
-  echo "running ginkgo"
-  run_ginkgo
-  # to fail the script if ginkgo failed
-  EXIT_CODE=$?
-else
-  echo "running scripts/parser/main.go"
-  go run scripts/parser/main.go \
-    $BASEDIR/avalanchego-${VERSION}/output.yaml \
-    $CHAIN_ID $GENESIS_ADDRESS \
-    $BASEDIR/avalanchego-${VERSION}/avalanchego \
-    ${AVALANCHEGO_PLUGIN_DIR} \
-    "0.0.0.0:12342" \
-    "$BASEDIR/genesis.json"
-fi
+echo "running ginkgo"
+run_ginkgo
+# to fail the script if ginkgo failed
+EXIT_CODE=$?
```

Comment:
Is this intended to run the ginkgo tests and then the simulator as well?

Reply:
This exit code will only be overwritten if the simulator tests are enabled. Otherwise, only used for failing the run.sh script if ginkgo fails (in this case by default, ginkgo fails to launch subnet-evm in the local network).

So, only runs simulator when RUN_SIMULATOR=true